manifests/kustomization.yaml

```diff
@@ -2,5 +2,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
 - guestbook.yaml
-- guestbook-public.yaml
-- redis.yaml
\ No newline at end of file
+- redis.yaml
```

Comment:
do you need to commit this file? at least `gabriel-okteto` related values shouldn't be commited